src/dodal/devices/watsonmarlow323_pump.py

```diff
@@ -0,0 +1,52 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_rw
+
+
+class WatsonMarlow323PumpEnable(str, Enum):
+    DISABLED = "Disabled"
+    ENABLED = "Enabled"
+
+
+class WatsonMarlow323PumpDirection(str, Enum):
+    CLOCKWISE = "CW"
+    COUNTER_CLOCKWISE = "CCW"
+
+
+class WatsonMarlow323PumpState(str, Enum):
+    STOPPED = "STOP"
+    STARTED = "START"
+
+
+class WatsonMarlow323Pump(StandardReadable):
+    """Watson Marlow 323 Peristaltic Pump device"""
+
+    def __init__(self, prefix: str, name: str = "") -> None:
+        self.enabled = epics_signal_rw(
+            WatsonMarlow323PumpEnable,
+            prefix + "DISABLE",
+        )
+
+        self.direction = epics_signal_rw(
+            WatsonMarlow323PumpDirection,
+            read_pv=prefix + "INFO:DIR",
+            write_pv=prefix + "SET:DIR",
+        )
+        self.state = epics_signal_rw(
+            WatsonMarlow323PumpState,
+            read_pv=prefix + "INFO:RUN",
+            write_pv=prefix + "SET:RUN",
+        )
+        self.speed = epics_signal_rw(
+            float, read_pv=prefix + "INFO:SPD", write_pv=prefix + "SET:SPD"
+        )
+
+        self.set_readable_signals(
+            read=[self.state, self.speed, self.direction],
+            config=[
+                self.enabled,
+            ],
+        )
+
```

Comment:
nit: this could be done with the `add_children_as_readables` context, which is now the recommended way to do it [in the ophyd-async docs](https://blueskyproject.io/ophyd-async/main/how-to/make-a-simple-device.html#readable)